src/lib/db/DatabaseManager.ts

```diff
@@ -9,84 +9,159 @@ import { v4 as uuidv4 } from "uuid";
-    timestamp: new Date().toISOString(),
-  },
-];
+const seedData = {
+  categories: [
+    {
+      id: uuidv4(),
+      name: "Replace",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Barrier",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Distract",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Change Status",
+      timestamp: new Date().toISOString(),
+    },
+  ],
+  toolkit: [
+    {
+      id: uuidv4(),
+      name: "Listen to my favourite music",
+      categories: ["Replace", "Barrier"],
+      checked: false,
+      infoUrl: "https://google.com/music",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Watch TV",
+      categories: ["Distract"],
+      checked: false,
+      infoUrl: "https://google.com/tv",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Call a friend",
+      categories: ["Distract", "Change status"],
+      checked: false,
+      infoUrl: "https://example.com/call",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "See a friend",
+      categories: ["Distract", "Change status"],
+      checked: false,
+      infoUrl: "https://example.com/call",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+  ],
+};
 
 class DatabaseManager {
   dbInstance: RxDatabase | null = null;
 
   async initialiseDatabase() {
+    console.groupCollapsed("Initialising Database");
     if (!this.dbInstance) {
+      console.groupCollapsed("Creating New Database");
       this.dbInstance = await createRxDatabase({
         name: "database",
         //ignoreDuplicate: true,
         storage: getRxStorageDexie(),
       });
+      console.log("Database Created");
+      console.groupEnd();
 
+      console.groupCollapsed("Adding Collections");
       await this.dbInstance.addCollections({
         categories: { schema: categoriesSchema },
         mood_records: { schema: moodRecordSchema },
         toolkit_items: { schema: toolkitItemSchema },
       });
+      console.log("Collections Added");
+      console.groupEnd();
 
-      console.log("Database initialised");
-      await this.seedToolkitItems(); // Automatically seed data
+      console.groupCollapsed("Seeding Database");
+      await this.seedCategories();
+      await this.seedToolkitItems();
```

Comment:
Isn't there still a risk that this is creating a loop? I think the seed methods could simply access the database dbInstance directly, see below a suggestion.

Reply:
It almost certainly is; I've left them in place as fixing the problem was outside the scope of the branch. Added the logs to help find it later.